Tests/TuistKitTests/ImportFinder/ImportSourceCodeScannerTests.swift

```diff
@@ -103,6 +103,34 @@ final class ImportSourceCodeScannerTests: TuistUnitTestCase {
         XCTAssertEqual(imports, ["PackageDescription"])
     }
 
+    func test_whenSwiftWithDefaultImportWithComment() throws {
+        let code = """
+        ////        import PackageDescription
+
+        func a() { }
+        """
+        let imports = try subject.extractImports(
+            from: code,
+            language: .swift
+        )
+        XCTAssertEqual(imports, [])
+    }
+
+    func test_whenSwiftWithDefaultImportWithMultilineComment() throws {
+        let code = """
+        /*
+        import PackageDescription
+        */
+
+        func a() { }
+        """
+        let imports = try subject.extractImports(
+            from: code,
+            language: .swift
+        )
+        XCTAssertEqual(imports, [])
```

Comment:
```suggestion
        // Given
        let code = """
        /*
        import PackageDescription
        */

        func a() { }
        """
        
        // When
        let imports = try subject.extractImports(
            from: code,
            language: .swift
        )
        
        // Then
        XCTAssertEqual(imports, [])
```

nit: let's follow the `// Given // When // Then` pattern that we use across the codebase